script.module.resolveurl/lib/resolveurl/plugins/drkvid.py

```diff
@@ -23,8 +23,8 @@
 
 class DrkVidResolver(ResolveGeneric):
     name = 'DrkVid'
-    domains = ['drkvid.site', 'allviid.online','telvod.site','allviid.info']
-    pattern = r'(?://|\.)(.*?\.(?:drkvid|allviid|telvod)\.(?:site|online|info))/(?:embed-)?([0-9a-zA-Z]+)'
+    domains = ['drkvid.site', 'allviid.online','telvod.site','allviid.info', 'allviidss.buzz']
+    pattern = r'(?://|\.)(.*?\.(?:drkvid|allviid|telvod|allviidss)\.(?:site|online|info|buzz))/(?:embed-)?([0-9a-zA-Z]+)'
```

Comment:
Can be done better
`    pattern = r'(?://|\.)(.*?\.(?:drkvid|allviids*|telvod)\.(?:site|online|info|buzz))/(?:embed-)?([0-9a-zA-Z]+)'`